### PR TITLE
 string literals were added

### PR DESCRIPTION
**Pull request by cristian-programmer**
character strings were changed to character literals, which decreases the use of the concatenation operator + and gives the code more readability

### CODE REVIEW

**Comment by sleepy-monax** [comment]
Can you make cleaner and more atomic git commit because right now it's kind of a mess and it's really difficult to review, You said in you pull request message you've added some better string literals but you touch like every file in the repo and merge two times.

File: main.js [hunk 1]
```diff
@@ -5,24 +5,25 @@ function update_releases()
     fetch('https://api.github.com/repos/skiftOS/skift/releases')
```
Comment (sleepy-monax) [inline main.js:5]: Can you do a pull request with only these changes

**Review by cristian-programmer — COMMENTED** [review]
I already did the review, it's fine like that.

File: main.js (hunk 1 above)
Comment (cristian-programmer) [reply main.js:5]: yes :)

**Pull request merged**